kolibri/plugins/device/assets/src/views/ManageContentPage/ChannelListItem.vue

```diff
@@ -37,7 +37,7 @@
     </template>
 
     <template slot="meta">
-      <div v-if="inImportMode && onDevice" class="spec-ref-on-device">
+      <div v-if="inImportMode && onDevice" data-test="on-device">
```

Comment:
Interesting! I like this strategy much better than using a `class` with the `spec-ref-` prefix.

If we're going to start using this convention, I'd slightly prefer a more explicit name like `data-spec-ref` or `data-test-ref` because initially I thought `data-test` was leftover debugging code.